sovtokenfees/doc/Interface/README.md

```diff
@@ -421,13 +421,23 @@ Example:
 }
 ```
 
-Validation steps for this example:
+The pool performs the following validation for the given example:
 * doDynamicValidation for "adding new steward" nym (_`from indy-node's side`_);
     * making general role's authorization (_`from indy-node's side`_)
     * making fees specific validation, using metadata field (_`from plugin's side`_)
         * lookup through config state for getting amount of "add_new_steward" alias (_`from plugin's side`_)
         * making can_pay_fees validation (_`from plugin's side`_)
         
+### Notes:
+* The order of previous steps is very important. First of all SET_FEES is required, then - AUTH_RULE.
+* SET_FEES it's "updating" cmd. For example, if current fees is {A: 1, B: 2} then after sending SET_FEES cmd with {A: 42} map resulted fees will be looked as {A: 42, B: 2} 
```

Comment:
SET_FEES is "updating" transaction, so that it appends new aliases to the existing FEEs map (either adding or overriding aliases). For example, if current fees are {A: 1, B: 2} then after sending SET_FEES transaction with {A: 42, C:3}, the resulted map will look like {A: 42, B: 2, C:3}.